### ISSUE TITLE
avahi-discover: DeprecationWarning: Importing dbus.glib

### ISSUE THREAD

**Issue by simonbcn** [lathiat/avahi#246]
```
Arch Linux
avahi v0.7+18+g1b5f401-2 (installed from official repositories)
```
```
$ avahi-discover
/usr/bin/avahi-discover:45: DeprecationWarning: Importing dbus.glib to use the GLib main loop with dbus-python is deprecated.
Instead, use this sequence:

    from dbus.mainloop.glib import DBusGMainLoop

    DBusGMainLoop(set_as_default=True)

  import dbus.glib
....
```

**Comment by smcv** [comment]
This was fixed in #220, which was included in v0.8.

**Comment by evverx** [comment]
I assume it can be closed.
